src/20_Components/amp-dynamic-css-classes.html

```diff
@@ -0,0 +1,58 @@
+<!--
+  #### Introduction
```

Comment:
Can you add a introduction? For formatting reasons, please put the intro text into a separate comment: 

```
<!-- Introduction -->
<!-- Text ... -->
```